README.md

```diff
@@ -19,10 +19,10 @@ In order to use Salesforce SSO authentication, you **must** create a connected a
 1. On the top right, click the "New Connected App" button.
 1. Enter an app name, API name, and contact email of your choice.
 1. Click the "Enable OAuth Settings" toggle.
-1. Enter a Callback URL to allow Salesforce to redirect users back to your Pardot plugin settings page. The URL should look similar to: `https://[YourWordpressDomainHere]/wp-admin/options-general.php?page=pardot`.
-1. Add "Access Pardot Services (pardot_api)" and "Perform requests on your behalf at any time (refresh_token, offline_access)" to your selected OAuth scopes.
+1. Enter a Callback URL to allow Salesforce to redirect users back to your Account Engagement plugin settings page. The URL should look similar to: `https://[YourWordpressDomainHere]/wp-admin/options-general.php?page=pardot`.
+1. Add "Access Account Engagement Services (pardot_api)" and "Perform requests on your behalf at any time (refresh_token, offline_access)" to your selected OAuth scopes.
 1. Save your connected application. A new page will appear with the Consumer Key and Consumer Secret.
-1. Enter your Consumer Key, Consumer Secret, and Pardot Business Unit ID into the Pardot WordPress settings screen. To find the Pardot Business Unit ID, go to Salesforce Setup and enter "Pardot Account Setup" in the Quick Find box. Your Pardot Business Unit ID begins with "0Uv" and is 18 characters long. If you cannot access the Pardot Account Setup information, ask your Salesforce Administrator to provide you with the Pardot Business Unit ID.
+1. Enter your Consumer Key, Consumer Secret, and Account Engagement Business Unit ID into the Account Engagement WordPress settings screen. To find the Account Engagement Business Unit ID, go to Salesforce Setup and enter "Account Engagement Business Unit Setup" in the Quick Find box. Your Account Engagement Business Unit ID begins with "0Uv" and is 18 characters long. If you cannot access the Account Engagement Business Unit Setup information, ask your Salesforce Administrator to provide you with the Account Engagement Business Unit ID.
```

Comment:
Also thoughts on Jay's suggestion in #pardot-wordpress-plugin? My team could discuss it as well

Reply:
@pengchensfdc, can you clarify how what we have here differs from [the steps Jay outlined in Slack here](https://salesforce-internal.slack.com/archives/C018JLBCA07/p1695641978413729?thread_ts=1695641957.079769&cid=C018JLBCA07)? Best I can tell, these steps match what Jay mentioned, no?

Reply:
Read again it lgtm